systemtest/intake_async_test.go

```diff
@@ -26,19 +26,19 @@ import (
 )
 
 func TestIntakeAsync(t *testing.T) {
-	t.Run("semaphore_size_1", func(t *testing.T) {
+	t.Run("semaphore_size_5", func(t *testing.T) {
 		systemtest.CleanupElasticsearch(t)
-		// limit the maximum concurrent decoders to 1. This will allow to test
+		// limit the maximum concurrent decoders to 5. This will allow to test
 		// for the successful case (no concurrency and unsuccessful case).
-		srv := apmservertest.NewServerTB(t, "max_concurrent_decoders=1")
+		srv := apmservertest.NewServerTB(t, "-E", "apm-server.max_concurrent_decoders=5")
 
 		systemtest.SendBackendEventsAsyncPayload(t, srv.URL, `../testdata/intake-v2/errors.ndjson`)
 		// Ensure the 5 errors are ingested.
 		systemtest.Elasticsearch.ExpectMinDocs(t, 5, "logs-apm.error-*", nil)
 
 		// Send a request with a lot of events (~1920) and expect it to be processed
 		// without any errors.
-		systemtest.SendBackendEventsAsyncPayload(t, srv.URL, `../testdata/intake-v2/heavy.ndjson`)
+		systemtest.SendBackendEventsAsyncPayloadError(t, srv.URL, `../testdata/intake-v2/heavy.ndjson`)
```

Comment:
Why is this failing now? The comment above doesn't match.

Reply:
It's because the semaphore wasn't being set to what I intended, so it ended up with the default semaphore size, which is not that test's aim